internal/pkg/cli/svc_package.go

```diff
@@ -306,29 +293,32 @@ func (o *packageSvcOpts) configureClients() error {
 }
 
 type wkldCfnTemplates struct {
-	stack         string
-	configuration string
+	template   string
+	parameters string
 }
 
-// getSvcTemplates returns the CloudFormation stack's template and its parameters for the service.
-func (o *packageSvcOpts) getSvcTemplates(env *config.Environment) (*wkldCfnTemplates, error) {
+func (o *packageSvcOpts) getStackGenerator(env *config.Environment) (workloadStackGenerator, error) {
 	mft, err := workloadManifest(&workloadManifestInput{
 		name:         o.name,
 		appName:      o.appName,
 		envName:      o.envName,
 		interpolator: o.newInterpolator(o.appName, o.envName),
 		ws:           o.ws,
 		unmarshal:    o.unmarshal,
-		sess:         o.envSess,
 	})
 	if err != nil {
 		return nil, err
 	}
 	o.appliedManifest = mft
-	generator, err := o.newTplGenerator(o)
+	generator, err := o.newStackGenerator(o)
```

Comment:
nit:
```suggestion
	return o.newStackGenerator(o)
```

Reply:
resolved in https://github.com/aws/copilot-cli/pull/3820/commits/ea721cc75996dbf65858faa8b81e6da2e53878a4!